extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -0,0 +1,209 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import { MigrationWizardPage } from '../models/migrationWizardPage';
+import { MigrationStateModel, StateChangeEvent } from '../models/stateMachine';
+import { CreateIntegrationRuntimeDialog } from './createMigrationControllerDialog';
+import * as constants from '../models/strings';
+import * as os from 'os';
+
+export class IntergrationRuntimePage extends MigrationWizardPage {
+
+	private migrationControllerDropdown!: azdata.DropDownComponent;
+	private defaultSetupRadioButton!: azdata.RadioButtonComponent;
+	private customSetupRadioButton!: azdata.RadioButtonComponent;
+	private startSetupButton!: azdata.ButtonComponent;
+	private cancelSetupButton!: azdata.ButtonComponent;
+	private _connectionStatus!: azdata.TextComponent;
+	private createMigrationContainer!: azdata.FlexContainer;
+	private _view!: azdata.ModelView;
+
+	constructor(wizard: azdata.window.Wizard, migrationStateModel: MigrationStateModel) {
+		super(wizard, azdata.window.createWizardPage(constants.IR_PAGE_TITLE), migrationStateModel);
```

Comment:
Sai just added a telemetry name field for wizard pages - you should add that to all the pages here so they're easier to identify.

Reply:
I will do this in the future iterations as merging this branch with the main is giving me weird errors.